mkt/zadmin/templates/admin/base.html

```diff
@@ -73,15 +73,6 @@
                       </li>
                     </ul>
                   </li>
-                {% else %}
```

Comment:
Won't this mean there isn't a login link on admin pages?  (That wouldn't be the end of the world tbh)